packages/react-dev-utils/InterpolateHtmlPlugin.js

```diff
@@ -15,7 +15,6 @@
 // Learn more about creating plugins like this:
 // https://github.com/ampedandwired/html-webpack-plugin#events
 
-'use strict';
```

Comment:
```
/home/travis/build/facebookincubator/create-react-app/packages/react-dev-utils/InterpolateHtmlPlugin.js
  18:1  warning  'use strict' is unnecessary inside of modules  strict
```

I can ignore it, I was just listening to eslint. 😛 Was waiting to see if CI passed on node 4.